src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/SimpleRenameStrategy.cs

```diff
@@ -23,7 +23,7 @@ public override bool CanHandleRename(string oldFileName, string newFileName)
         {
             var oldNameBase = Path.GetFileNameWithoutExtension(oldFileName);
             var newNameBase = Path.GetFileNameWithoutExtension(newFileName);
-            return SyntaxFacts.IsValidIdentifier(oldNameBase) && SyntaxFacts.IsValidIdentifier(newNameBase);
+            return SyntaxFacts.IsValidIdentifier(oldNameBase) && SyntaxFacts.IsValidIdentifier(newNameBase) && !Equals(Path.GetFileName(oldNameBase), Path.GetFileName(newNameBase));
         }
```

Comment:
And this probably just represents the case of the `<Compile Include="Foo.cs" />' - and should this result in a rename for VB which is also case-insenstive?